packages/chrome/src/styled/sheet/StyledSheetFooterItem.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet_footer_item';
+
+export const StyledSheetFooterItem = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  margin: auto 10px;
```

Comment:
Similar https://github.com/zendeskgarden/react-components/pull/1231/files?authenticity_token=ocFDB1GmyeZM0qKO%2B%2B%2BaueEsdBPb4UU%2F%2FUUGDkPrbzizvqjKB%2FQzJUVHh0bjsIMD1%2FkviL9azLOv18Ws9Ws0vA%3D%3D&file-filters%5B%5D=.lock&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r741539973

Reply:
According to the `Sheet` design spec and like `StyledBody`, `FooterItem`s have a 10px margin. Do you think this should be a scalar value based on another value?